### DISCUSSION TITLE
pinecone init project problem

### DISCUSSION THREAD

**Discussion by jaebeompark** [FlowiseAI/Flowise#187] [Q&A]
Thank you for creating flowise.

I'm creating a Q&A for a pdf document, and I'm seeing this error message.
values like pinecone api key, environment and index are all correct.

What should I be looking at?

[ErrorWithoutStackTrace: PineconeClient: Error getting project name: SyntaxError: Unexpected token A in JSON at position 0]

**Comment by wsligter** [comment]
Same issue here. Did you find out how to solve this?

**Reply by TummenUppGeInteUpp** [reply]
I have the sam issue, it would be fantastic with a smal guide to solve this error.

**Comment by TummenUppGeInteUpp** [comment]
i solved that issue with changing the Environment: from westus to Environmentasia-southeast1-gcp-free

**Comment by govind-kumarr** [comment]
![Screenshot (601)](https://github.com/FlowiseAI/Flowise/assets/106414173/75101344-cb34-4405-bdf5-0dacd0a9807c)
does it looks like this
